de.json

```
@@ -641,36 +641,36 @@
         "FullBody.AdjustAvatarInstructions" : "Passen Sie den Avatar-Körper-Versatz so an, dass er am besten zur Referenz passt",
-        "FullBody.FineTuneCalibrationInstructions" : "Überprüfen und feinjustieren der Kalibierung durch Ausrichtung der blauen Referenzen",
+        "FullBody.FineTuneCalibrationInstructions" : "Überprüfen und feinjustieren der Kalibierung durch Ausrichtung der blauen Referenzkugeln",
 
         "FullBody.CalibrateTrackersHeader" : "Tracker kalibrieren",
         "FullBody.CalibrateTrackersInstructions" : "Greifen und Positionieren Sie die blauen Körperteile relativ zu den Trackern um ihren Versatz zu justieren. Diese sollten den Postionen der Tracker auf Ihrem Körper entsprechen.\nSie können die Kalibrierung jederzeit später justieren, indem Sie den Kalibrator wieder starten.",
         "FullBody.UseSymmetry" : "Symmetrie verwenden",
         "FullBody.ShowBodyOverlay" : "Körperüberlagerung anzeigen",
-        "FullBody.HeightCompensationInstructions" : "Falls das Mannequin zu groß ist (sichtbar durch gekrümmte Haltung) oder zu klein ist, nutzen Sie die Höhenkompensation zur genauen Ausrichtung.",
+        "FullBody.HeightCompensationInstructions" : "Falls der Mannequin zu groß ist (erkennbar durch gekrümmte Haltung) oder zu klein ist, nutzen Sie die Höhenkompensation zur genauen Ausrichtung.",
         "FullBody.RecalibratePrompt" : "Nicht funtionierende or fehlende Tracker?",
         "FullBody.Recalibrate" : "Wiederholen Sie die T-Posen-Kalibrierung",
-        "FullBody.CalibrateAvatarPrompt" : "Ist es erforderlich die Avatar-Ausrichtung anzupassen?",
+        "FullBody.CalibrateAvatarPrompt" : "Muss die Avatar Ausrichtung angepasst werden?",
         "FullBody.CalibrateAvatar" : "Meinen Avatar kalibrieren",
         "FullBody.FinishPrompt" : "Alles OK?",
         "FullBody.FinishCalibration" : "Ganzkörper-Kalibrierung abschließen",
 
         "FullBody.CalibrateAvatarTitle" : "Avatar kalibrieren",
         "FullBody.CalibrateAvatarInstructions" : "Stellen Sie sicher, dass alle Körperteile zugeordnet sind und bewegen Sie anschließend die blauen Kugeln, um den Avatar Ihreren Körper-Referenzen anzupassen.",
-        "FullBody.ShowReferenceOverlay" : "Referenz-Überlagerug zeigen",
+        "FullBody.ShowReferenceOverlay" : "Referenz-Überlagerung zeigen",
         "FullBody.ResetAvatarPrompt" : "Avatar verdreht oder ohne Tracking?",
         "FullBody.ResetAvatar" : "Zuordnung zurücksetzen",
-        "FullBody.AvatarHeightCompensationInstructions" : "Falls der Avatar zu groß ist (sichtbar durch gekrümmte Haltung) oder zu klein ist, können Sie die Höhe mit der Höhen Kompensation anpassen, so dass er besser zu Ihrer Höhe passt.",
+        "FullBody.AvatarHeightCompensationInstructions" : "Falls der Avatar zu groß ist (erkennbar durch gekrümmte Haltung) oder zu klein ist, können Sie die Höhe mit der Höhen Kompensation anpassen, so dass er besser zu Ihrer Höhe passt.",
```

A small fix from me but else yes
```suggestion
        "FullBody.AvatarHeightCompensationInstructions" : "Falls der Avatar zu groß ist (erkennbar durch gekrümmte Haltung) oder zu klein ist, können Sie die Höhe mit der Höhen-Kompensation anpassen, so dass er besser zu Ihrer Höhe passt.",
```